.github/workflows/windows.yml

```diff
@@ -88,34 +86,17 @@ jobs:
           # For running Paddle frontend unit tests
           python3 -m pip install -r ${{ env.OPENVINO_REPO }}/src/frontends/paddle/tests/requirements.txt
 
-      - name: Install MO dependencies
-        run: |
-          python3 -m pip install -r ${{ env.OPENVINO_REPO }}/tools/mo/requirements_mxnet.txt `
-          -r ${{ env.OPENVINO_REPO }}/tools/mo/requirements_caffe.txt `
-          -r ${{ env.OPENVINO_REPO }}/tools/mo/requirements_kaldi.txt `
-          -r ${{ env.OPENVINO_REPO }}/tools/mo/requirements_onnx.txt `
-          -r ${{ env.OPENVINO_REPO }}/tools/mo/requirements_tf2.txt `
-          -r ${{ env.OPENVINO_REPO }}/tools/mo/requirements_dev.txt
-
       - name: Install build dependencies
         run: |
           choco install --no-progress ninja
           choco install --no-progress shellcheck
```

Comment:
`shellcheck` is not required, it's enough to test shell script only on Linux with `shellcheck`